src/pages/search/search.ts

```diff
@@ -47,6 +49,16 @@ export class SearchPage {
       return;
     }
 
+    if (!this.network.type || this.network.type == 'unknown' || this.network.type == 'none') {
+      let toast = this.toastCtrl.create({
+        message: 'Please enable a data connection to search',
+        duration: 2000,
+        position: 'bottom'
+      });
+      toast.present();
+      return
+    }
+
```

Comment:
Looks like this is repeated piece of code… can it be separated out?

Reply:
It's very similar to one other place. I'm inclined to say it's not worth separating out. Do you feel strongly about it?

Reply:
Feel strong… no.

But… I see us using it more than just here.

Reply:
How about I go ahead and merge this, and if/when we use this elsewhere we can be more clever about it?